radix-engine/src/system/client_api_impl.rs

```diff
@@ -670,6 +671,7 @@ impl<'g, 's, W> ClientCostingApi<RuntimeError> for Kernel<'g, 's, W>
 where
     W: WasmEngine,
 {
+    #[trace_resources(log=units)]
```

Comment:
Suggest tracing for all methods in ClientApi impl, and may be give it a group name?